AgentDiscoveries-Backend/src/test/java/org/softwire/training/utils/PagerTest.java

```diff
@@ -0,0 +1,77 @@
+package org.softwire.training.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PagerTest {
+
+    Pager pager;
+
+    @BeforeEach
+    void setUp() {
+        pager = new Pager(100, 2, 10);
+    }
+
+    @Test
+    void getTotalItems() {
+        int totalItems = pager.getTotalItems();
+
+        assertEquals(100, totalItems);
+    }
+
+    @Test
+    void getCurrentPage() {
+        int currentPage = pager.getCurrentPage();
+
+        assertEquals(2, currentPage);
+    }
+
+    @Test
+    void getPageSize() {
+        int pageSize = pager.getPageSize();
+
+        assertEquals(10, pageSize);
+    }
+
+    @Test
+    void calculatesNumOfPages() {
+        int totalPages = pager.getTotalPages();
+
+        assertEquals(10, totalPages);
+    }
+
+    @Test
+    void calculatesStartPage() {
```

Comment:
I don't think this test actually tests the general case, we only test the edge case at the moment.

Can we also make the test name describe what the start page should be? (i.e. should be five before the current page)

Writing good tests is a fine art! Sorry if I sound a bit picky...